resources/src/jvmMain/kotlin/dev/icerock/moko/resources/PluralsResource.kt

```diff
@@ -0,0 +1,6 @@
+package dev.icerock.moko.resources
+
+actual class PluralsResource(
+    val key: String,
+    val numberFormat: List<Pair<Double, String>>
+)
```

Comment:
logic of resolving required string should be here i think

Reply:
I'm not sure what you mean 🤔 , currently the resolving logic is in the `PluralStringDesc`, because it contains the `number` parameter

Reply:
in my branch i move logic as i describe